scripts/check-expo-dependencies.js

```diff
@@ -0,0 +1,39 @@
+module.exports = async ({ context, github, fetch, prTitle }) => {
+  // eslint-disable-next-line @typescript-eslint/no-var-requires
+  const expoVersion = require('../package.json').dependencies.expo
+  const expoVersionSanitized = expoVersion.replaceAll('~', '')
```

Comment:
are you sure that tilde is the only possible symbol? maybe it is now, but maybe expo at some point may decide to use other version specifiers too, e.g. ^